### DISCUSSION TITLE
revit by itwin snapshot

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#4852] [Q&A] [answered]
i setup the itwin snapshot APP (https://www.itwinjs.org/learning/tutorials/itwin-snapshot-app/)
 and install the revit by the imodel connectors ui
but tell me must install revit and not change the file to bim.

 
![123](https://user-images.githubusercontent.com/18022720/209806673-e6b3b31c-33b5-443f-b9a6-b5390387565f.png)

**Comment by arnab-git-hub** [comment, marked as answer]
When you click on the download button as shown in the image, it will install the free Revit Connector to your system, not the Revit software itself.

Can you please provide some more details?

**Reply by jffmarker** [reply]
I'll just re-affirm that yes, the iTwin Snapshot App refers to the free Revit Connector that it will download/install/auto-update, _not_ the design application. Please click the download icon and try again.
![image](https://user-images.githubusercontent.com/8006049/211015086-4806851f-4f41-466d-98ea-9edc26a8bbc3.png)
